server/proxy/pf_config.c

```diff
@@ -632,6 +669,8 @@ BOOL pf_server_config_dump(const char* file)
 		goto fail;
 	if (IniFile_SetKeyValueString(ini, section_target, key_target_fixed, bool_str_true) < 0)
 		goto fail;
+	if (IniFile_SetKeyValueString(ini, section_target, key_target_tls_seclevel, "1") < 0)
```

Comment:
Shouldn't we use `IniFile_SetKeyValueInt` here to indicate that this setting is an integer?

Reply:
yes, we can use that too.